README.md

```diff
@@ -1,2 +1,69 @@
-# gcp-pubsub-deadletter-logger
-Log dead letter events to Google Cloud Logging
+# GCP Pub/Sub dead-letter logger
+[![Go Report Card](https://goreportcard.com/badge/github.com/YourSurpriseCom/gcp-pubsub-deadletter-logger)](https://goreportcard.com/report/github.com/YourSurpriseCom/gcp-pubsub-deadletter-logger) 
+![workflow ci](https://github.com/YourSurpriseCom/gcp-pubsub-deadletter-logger/actions/workflows/ci.yml/badge.svg)
+![workflow release ](https://github.com/YourSurpriseCom/gcp-pubsub-deadletter-logger/actions/workflows/release.yml/badge.svg)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+## Overview
+This application is used to log dead-letter events to Google Cloud Logging. 
+
+### Example usage
+The BigQuery subscription can sent every event directly to a BigQuery table. 
+When an event cannot be added to the BigQuery table, the error is added to the event message and sent to the dead-letter topic.
```

Comment:
There's nothing BigQuery specific about this application, right? This can just be used for _any_ dead letter topic, right?

Reply:
No, thats true. But the BigQuery subscription is one of the subscription where nog logs are created when something goes wrong. Thats why i used this as an example